src/utils/jira.py

```diff
@@ -126,6 +129,27 @@ def get_blocks(jira_client: jira.client.JIRA, issue: jira.resources.Issue):
     return blocks
 
 
+def get_children(jira_client: jira.client.JIRA, issue: jira.resources.Issue):
+    link_field_id = issue.raw["Context"]["Field Ids"]["Parent Link"].split("_")[-1]
+
+    query = (
+        f"cf[{link_field_id}]={issue.key} and resolution=Unresolved ORDER BY rank ASC"
+    )
+    # print("    ?", query)
+    # JIRA can be flaky, so retry a few times before failing
```

Comment:
Can you move that bit in its own func, and call the new func from `query_issues()` as well?

Reply:
Done in b4f537f.